setup.py

```diff
@@ -1,14 +1,10 @@
 #!/usr/bin/env python
-# -*- coding: utf-8 -*-
 
```

Comment:
1) there are no non-ASCII characters in this file
2) Python 3 defautls to utf-8 anyway
3) PEP 8 disallows the usage of an encoding declaration when it's unnecessary